packages/autocomplete-core/src/getDefaultProps.ts

```diff
@@ -60,13 +60,27 @@ export function getDefaultProps<TItem>(
         [...getSourcesFromPlugins, props.getSources].map((getSources) =>
           getNormalizedSources(getSources!, options)
         )
-      ).then((nested) =>
-        // same as `nested.flat()`
-        nested.reduce((acc, array) => {
-          acc = acc.concat(array);
-          return acc;
-        }, [])
-      );
+      )
+        .then((nested) =>
+          // same as `nested.flat()`
+          nested.reduce((acc, array) => {
```

Comment:
Why do we need to flatten sources here? (Does it reveal a data structure problem prior to this instruction?)

Reply:
No big deal, just to make the next iteration easier.